packages/appsync-modelgen-plugin/src/__tests__/visitors/__snapshots__/appsync-java-visitor.test.ts.snap

```diff
@@ -39,29 +39,29 @@ public final class customClaim implements Model {
   public String getId() {
       return id;
   }
-  
```

Comment:
It looks like your editor made auto-formatting changes to the snapshots. Could you revert these changes?

Reply:
I think I got it fixed - waiting on tests to run